kubernetes/windows/main.ps1

```diff
@@ -151,10 +115,16 @@ function Set-CloudSpecificApplicationInsightsConfig {
         }
         "usnat" {
             Write-Host "Set-CloudSpecificApplicationInsightsConfig: Setting Application Insights configuration for USNat Cloud"
+            Set-ProcessAndMachineEnvVariables "APPLICATIONINSIGHTS_AUTH" "YTk5NTlkNDYtYzE3Zi0xZDYxLWJhODgtZWU3NDFjMGI3MTliCg=="
+            # IngestionEndpoint: usnateast-0.in.applicationinsights.azure.eaglex.ic.gov
+            Set-ProcessAndMachineEnvVariables "APPLICATIONINSIGHTS_ENDPOINT" "https://dc.applicationinsights.azure.eaglex.ic.gov/v2/track"
             Set-AirgapCloudSpecificApplicationInsightsConfig
```

Comment:
we can remove this Set-AirgapCloudSpecificApplicationInsightsConfig and since we dont need anything special soverign clouds.